docs/Oracle Jobs/service-agreement-examples.md

```diff
@@ -3,29 +3,16 @@ layout: nodes.liquid
 date: Last Modified
 title: "Service Agreement Examples"
 permalink: "docs/service-agreement-examples/"
-hidden: true
+hidden: false
```

Comment:
Was this intentional?

Reply:
I think that's a bug in UI. The hidden status wasn't changed